dask_expr/io/tests/test_io.py

```diff
@@ -282,12 +282,16 @@ def test_to_parquet(tmpdir, write_metadata_file):
         df2.to_parquet(tmpdir, overwrite=True)
 
 
-def test_combine_similar(tmpdir):
+@pytest.mark.parametrize(
+    "fmt,read_func,read_cls",
```

Comment:
Personally, I like spaces in between these arguments, but no blocker

Reply:
Didn't even realize I was allowed to leave spaces here, but that makes sense :)